client/layout.html

```diff
@@ -1,40 +1,15 @@
 <head>
-  <title>Make Lunch</title>
+  <script>
+    UPLOADCARE_PUBLIC_KEY = '15127b47da228b9b2d87'
```

Comment:
I know it's public, but should this be hard-coded rather than in `settings.json`